tests/gold_tests/proxy_protocol/proxy_protocol.test.py

```diff
@@ -17,36 +17,37 @@
 #  limitations under the License.
 
 import os
-from ports import get_port
-import sys
 
 Test.Summary = 'Test PROXY Protocol'
-Test.SkipUnless(
-    Condition.HasCurlOption("--haproxy-protocol")
-)
 Test.ContinueOnFail = True
 
 
-class ProxyProtocolTest:
+class ProxyProtocolInTest:
     """Test that ATS can receive Proxy Protocol."""
 
+    replay_file = "replay/proxy_protocol_in.replay.yaml"
+
     def __init__(self):
         self.setupOriginServer()
+        self.setupPVOriginServer()
         self.setupTS()
 
     def setupOriginServer(self):
         self.httpbin = Test.MakeHttpBinServer("httpbin")
```

Comment:
Do we still need the `httpbin` backend?

Reply:
Nope. That's a good catch. I cleaned up all usages of `httpbin`.